src/app/models/Response.ts

```diff
@@ -1,7 +1,10 @@
+import { Question } from '../../app/models/Question';
+
 export interface Response {
   id: number;
   responderId: number;
   questionId: number;
   body: string;
   creationDate: string;
+  question: Question[];
```

Comment:
why array of questions?

Reply:
I just checked. It shouldn't be an array